doc/sphinxman/source/scf.rst

```diff
@@ -745,6 +751,45 @@ To avoid this, either set |scf__df_basis_scf| to an auxiliary
 basis set defined for all atoms in the system, or set |scf__df_scf_guess|
 to false, which disables this acceleration entirely.
 
+COSX Exchange
+~~~~~~~~~~~~~
+
+The semi-numerical COSX algorithm described in [Neese:2009:98]_ evaluates
+two-electron ERIs analytically over one electron coordinate and numerically
+over the other electron coordinate, and belongs to the family of pseudospectral 
+methods originally suggested by Friesner. In COSX, numerical integration is performed on standard
```

Comment:
I still don't agree with mentioning pseudospectral methods and Friesner. Non-experts have no clue who Friesner is and would be better served by reading the Neese paper. Experts already know this. For this reason, in neither case are we helping people who want to learn about the method or people in the "what-exactly-have-I-run-and-why stage."